src/utils/createMerkleTree.ts

```
@@ -0,0 +1,25 @@
+import { StandardMerkleTree } from "@openzeppelin/merkle-tree";
+import * as fs from "fs";
+
+const createMerkleTree = async () => {
+  let data;
+
+  // `merkle_data.json` is the output of the `getStakerData` script in the `token-subgraphs` repo
+  if (fs.existsSync("output/merkle_data.json")) {
+    data = JSON.parse(fs.readFileSync("output/merkle_data.json", "utf-8"));
+  } else {
+    // Temporarily stub data in place of file read
+    data = [
+      ["0x123", 100n, 100n],
+    ];
```

we should probably throw here if file is not present. this would make it possible to run this whole flow if we forget to add the file, which shouldn't be the case. + this dummy data is not really doing anything useful for us anyway